src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/EdgeCasesTests.cs

```diff
@@ -105,6 +105,7 @@ public static void ZeroLengthContent_RoundTrip()
                 }
                 catch (CryptographicException) when (PlatformDetection.IsFullFramework) // Expected on full FX
                 {
+                    return;
```

Comment:
Alright, I marked it skip. Also renamed #18724 and tagged it netfx-port-consider. Better than opening an actual bug (which would get closed )